spec/board_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require_relative '../lib/board.rb'
+
+RSpec.describe Board do
+  let(:board) { Board.new }
+
+  describe '#Board' do
+    it 'shows cell interface' do
+      expect(board.cell).to eql([[1, 2, 3], [4, 5, 6], [7, 8, 9]])
+    end
+  end
+
+  describe '#move' do
```

Comment:
Could we wrap this inside only one `describe` block instead? :thinking: We could use the `context` block if we want to describe different scenarios.

```suggestion
  describe '#move' do
    context 'when moved element is on a different cell that is supposed to be' do
      it 'returns false on win_game  ' do
        expect(board.move('o', 4)).to eql([false, 1, [[1, 2, 3], ['o', 5, 6], [7, 8, 9]]])
      end
    end
    context 'when moved element is on a correct cell that is supposed to be' do
      it 'returns true on win_game ' do
        board_win = Board.new([[1, 'o', 'o'], [4, 5, 6], [7, 8, 9]])
        expect(board_win.move('o', 1)).to eql([true, 1, [%w[o o o], [4, 5, 6], [7, 8, 9]]])
      end
    end
  end
```